### PR TITLE
Feature/GearsScaffod, GearsModalBottomSheetLayout, GearsSideDialog

### PR DESCRIPTION
**Pull request by agungkq**
## NEW
1. [Bottom Sheet Component](https://app.clickup.com/t/3d90qeq)

### CODE REVIEW

File: dependencies.gradle [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ext {
-    composeUiVersion = "1.0.4"
+    composeUiVersion = "1.3.1"
 
     plugins = [
-            android_gradle: "com.android.tools.build:gradle:7.0.2",
-            kotlin        : "org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.31"
+            android_gradle: "com.android.tools.build:gradle:7.0.4",
+            kotlin        : "org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.10"
```
Comment (kristokhosim) [inline dependencies.gradle]: update versi `gradle-plugin` nya ke `1.7.20` @agungkq

File: gears-ui/build.gradle [hunk 2]
```diff
@@ -36,7 +36,7 @@ android {
     }
 
     composeOptions {
-        kotlinCompilerExtensionVersion '1.0.4'
+        kotlinCompilerExtensionVersion '1.3.1'
```
Comment (kristokhosim) [inline gears-ui/build.gradle]: update `compiler compose` ke `1.3.2`

**Review by kristokhosim — COMMENTED** [review]
tolong tambah lib `compose foundation `juga ya kalo memang blm ada @agungkq

File: gears-ui/build.gradle (hunk 2 above)
Comment (agungkq) [reply gears-ui/build.gradle]: done per commit [e5448d7](https://github.com/oddy-dev/oddy-gears-ui/pull/39/commits/e5448d71bb197da5818a1c985380d0e12b7fc146)

**Pull request merged**